### PR TITLE
Fixing Job dependencies for Release Action

### PR DESCRIPTION
**Pull request by nniehoff**
This PR Fixes the broken dependencies from the Release action.  🤞 1.2.6 can use this action.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -119,8 +119,8 @@ jobs:
 
   slack-notify:
     needs:
-      - "publish_gh"
-      - "publish_pypi"
+      - "publish_python"
+      - "container-publish"
```
Comment (glennmatthews) [inline .github/workflows/release.yml:123]: Can we change these two names to be consistent while we're in here? `publish_python` and `publish_docker` perhaps?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -43,7 +43,7 @@ jobs:
   container-publish:
     runs-on: "ubuntu-20.04"
     needs:
-      - "integration-test"
+      - "publish_python"
```
[the remark this answers is not in the document]
Comment (nniehoff) [reply .github/workflows/release.yml:46]: I was thinking the docker build used the artifact from pypi but your right it doesn't they are independent jobs.  Fixed

**Comment by nniehoff** [comment]
I have looked high and low for this, unfortunately googling "github action linters" returns every linter under the sun except linters for github actions, but I finally found [https://rhysd.github.io/actionlint/](https://rhysd.github.io/actionlint/) to actually lint GitHub workflow actions!  release.yml now validates, I tested the previous version and it did catch the errors.

**Review by glennmatthews — APPROVED** [review]
Thank you!

**Pull request merged**